### PR TITLE
is core

### PR DESCRIPTION
**Pull request by aromakh**

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -350,6 +350,25 @@ paths:
         "200":
           description: "ok"
       x-swagger-router-controller: "Play"
+  /brick/setCoreLibrary/{brickId}/{isCore}:
+    put:
+      tags:
+      - "brick"
+      summary: "Marks a single notification as read."
```
Comment (william-gooch) [inline api/swagger.yaml]: Should be something like "Sets whether a brick is in the core library or not."

File: service/BrickService.js [hunk 2]
```diff
@@ -394,6 +394,20 @@ exports.brickReviewIdPOST = function(id) {
 }
 
 
+/**
+ * Marks a single notification as read.
```
Comment (william-gooch) [inline service/BrickService.js]: Similar to other comment.

**Review by william-gooch — CHANGES_REQUESTED** [review]

File: api/swagger.yaml (hunk 1 above)
Comment (aromakh) [reply api/swagger.yaml]: @william-gooch  done

File: service/BrickService.js (hunk 2 above)
Comment (aromakh) [reply service/BrickService.js]: @william-gooch  done

**Pull request merged**